build.gradle

```diff
@@ -84,6 +88,7 @@ dependencies {
     testCompile'com.opentable.components:otj-pg-embedded:0.9.0'
     testCompile 'org.xerial:sqlite-jdbc:3.20.0'
     testCompile 'com.oracle:ojdbc6:12.1.0.1-atlassian-hosted'
+    testCompile group: 'com.microsoft.sqlserver', name: 'mssql-jdbc', version: '6.2.1.jre8'
```

Comment:
Please change driver version to jre7 (at least), because Exposed targets Java 6 and tests runs on Java 7 sometime.

Reply:
I switched to JRE 7